lib/site_encrypt/certbot.ex

```diff
@@ -1,4 +1,7 @@
 defmodule SiteEncrypt.Certbot do
+  @type https_keys :: {:ok, [keyfile: String.t(), certfile: String.t(), cacertfile: String.t()]}
```

Comment:
I think it would be better to drop the `:ok` and tuple from this spec and just have the inner keyword list.